packages/commonwealth/server/models/snapshot_spaces.ts

```diff
@@ -0,0 +1,40 @@
+import * as Sequelize from 'sequelize';
+import { DataTypes } from 'sequelize';
+import { ModelStatic, ModelInstance } from './types';
+
+export type SnapshotSpaceAttributes = {
+  snapshot_space: string;
+  created_at?: Date;
+  updated_at?: Date;
+};
+
+export type SnapshotSpaceInstance = ModelInstance<SnapshotSpaceAttributes>;
+
+export type SnapshotSpaceModelStatic = ModelStatic<SnapshotSpaceInstance>;
+
+export default (
+  sequelize: Sequelize.Sequelize,
+  dataTypes: typeof DataTypes
+): SnapshotSpaceModelStatic => {
+  const SnapshotSpaces = <SnapshotSpaceModelStatic>sequelize.define(
+    'SnapshotSpaces',
+    {
+      snapshot_space: {
+        type: Sequelize.STRING,
+        primaryKey: true,
+        allowNull: false,
+      },
+      created_at: { type: Sequelize.DATE, allowNull: false },
+      updated_at: { type: Sequelize.DATE, allowNull: false },
+    },
+    {
+      tableName: 'SnapshotSpaces',
+      timestamps: true,
+      underscored: true,
+      createdAt: 'created_at',
+      updatedAt: 'updated_at',
+    }
+  );
+
+  return SnapshotSpaces;
```

Comment:
We may want an association defined explicitly (+ relevant attributes) if we're ever going to e.g. query all proposals in a space.